modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -469,8 +469,12 @@ export default function EditNotificationTemplate({
 														},
 													})
 												}
-												placeholder={Liferay.Language.get(
-													'use-terms-to-configure-recipients-for-this-notifications'
+												placeholder={Liferay.Util.sub(
+													Liferay.Language.get(
+														'use-terms-to-configure-recipients-for-this-notifications-x'
+													),
+													'[%ENTRY_CREATOR%], [%BUYER_NAME%]',
+													'.'
```

Comment:
Hi @danvb112. I'm sending this code to make an adjustment in a previous PR that [Brian asked me to do](https://github.com/brianchandotcom/liferay-portal/pull/124638).
Thanks.